android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -130,11 +134,11 @@ public Activity getActivity() {
         return registrar != null ? registrar.activity() : activity;
     }
 
-    private @Nullable PurchasesFlutterActivity getActivityFragment() {
+    private @Nullable FlutterFragmentActivity getActivityFragment() {
```

Comment:
Let's update the docs and put this in the release notes for the next beta :)

Reply:
I believe our only docs for this beta currently live in the release notes right? Will update those when I make the release